src/parser/parser.yy

```diff
@@ -401,8 +411,13 @@ unreserved_keyword
     | KW_SHORTEST           { $$ = new std::string("shortest"); }
     | KW_COUNT_DISTINCT     { $$ = new std::string("count_distinct"); }
     | KW_CONTAINS           { $$ = new std::string("contains"); }
+    | KW_NOT_CONTAINS       { $$ = new std::string("not contains"); }
     | KW_STARTS             { $$ = new std::string("starts"); }
+    | KW_STARTS_WITH        { $$ = new std::string("starts with"); }
+    | KW_NOT_STARTS_WITH    { $$ = new std::string("not starts with"); }
     | KW_ENDS               { $$ = new std::string("ends"); }
+    | KW_ENDS_WITH          { $$ = new std::string("ends with"); }
+    | KW_NOT_ENDS_WITH      { $$ = new std::string("not ends with"); }
```

Comment:
We don't have to handle these.

Reply:
but when I commit code it report some warning message .